srcpkgs/dlib/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'dlib'
+pkgname=dlib
+version=19.18
+revision=1
+build_style=cmake
+short_desc="Modern C++ toolkit containing machine learning algorithms and tools"
+maintainer="Felix Hanley <felix@userspace.com.au>"
+license="BSL-1.0"
+homepage="http://dlib.net"
+distfiles="https://github.com/davisking/${pkgname}/archive/v${version}.tar.gz"
+checksum=e9a8540ad0ff84eda92dea7127cd7d078457caf7ced64400c25c9a0b8c96e242
+
+makedepends="$(vopt_if openblas openblas-devel) $(vopt_if x11 libX11-devel)"
+build_options="openblas x11"
+desc_option_openblas="Enable OpenBlas for performance"
+desc_option_x11="Enable X11 for GUI components"
+build_options_default="openblas x11"
+configure_args=Release
```

Comment:
please move `configure_args` and `makedepends` (in that order) below `build_style`